src/ducky.py

```diff
@@ -70,16 +49,10 @@ def make_random(cls):
     def generate(cls, options: Optional[DuckRequest] = None):
         """Generate a duck from the provided request, else generate a random one."""
         options = options.dict() if options else cls.make_random()
+        colors = options["colors"]
```

Comment:
This won't work because that's not how the data is provided by the API request, it should remain in the form it currently is using multiple fields for the body, wing, etc.

Reply:
Right, I will fix that.